### DISCUSSION TITLE
请问如何将参考文献改为单倍行距？

### DISCUSSION THREAD

**Discussion by hotpot-fish** [sjtug/SJTUThesis#961] [提问] [answered]
图书馆专家审核意见中： 参考文件排版不规范： **行距为单倍行距**

模板中setup.tex  109行\setlength{\bibitemsep}{0pt} 已经设置成0pt，行距仍比word中五号字体单倍行距要宽

**Comment by LogCreative** [comment, marked as answer]
```latex
{
\linespread{1}\selectfont
% 参考文献
\printbibliography[heading=bibintoc]
}
```
但事实上 Word 模板由于开了**文档网格**，其行距并不是准确的单倍行距，而是根据网格间距而定。本模板使用 `\fixedlineskip{20bp}` 固定间距对于小四字的参考文献是合适的。

若需要与 Word 模板基本完全一致，需要同时定义参考文献字号为五号，并设定为固定间距 15.6 磅：
```latex
{
% 文献表字体
\renewcommand{\bibfont}{\zihao{5}}
% 设定固定间距
\fixedlineskip{15.6bp}
% 参考文献
\printbibliography[heading=bibintoc]
}
```

**Reply by hover03** [reply]
![image](https://github.com/user-attachments/assets/ffc2ecb5-eacd-4a3d-893f-461d25436f4a)
这句会报错诶【o(╥﹏╥)o】
2023 XeLatex

**Reply by LogCreative** [reply]
试一试 `\setbaselineskip` 呢，请确认一下是否使用的是 TeX Live 2023。

**Comment by hover03** [comment]
确实是使用2023，改成\setbaselineskip可以了，非常感谢！！！！！！！！！
